packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitTrinketTracker.tsx

```diff
@@ -49,8 +54,21 @@ export const UnitTrinketTracker = (props: IUnitFrameRenderData) => {
     '201811',
     '205782',
     '205712',
+    '209767', // Verdant Asp Sigil
+    '209347', // Verdant Glad Sigil
   ]; // Spell: 195756
 
+  const isHealer = [
```

Comment:
i remember we had one of these somewhere else.. might be good to extract into a common util